_includes/head.html

```diff
@@ -2,6 +2,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" type="image/png" href="imgs/favicon.png"/>
```

Comment:
You forgot to fix the URL here, it should be `href="{{ "/imgs/favicon.png" | relative_url }}"` or something similar. Everything else looks nice.

Reply:
Oj! I'll fix. :-)

Reply:
So I fixed this in my repo, but for some reason this doesn't get picked up in the PR here.

Reply:
I think you forgot to push it to GitHub, because this is exactly like what you have in your fork https://github.com/jeremiah/pelux.io/commit/0ff15da652a73840d307a1f4ef046aeca67b6252